### DISCUSSION TITLE
How to update an object in prisma

### DISCUSSION THREAD

**Discussion by MincePie** [prisma/prisma#17075] [Q&A]
I am trying to follow [this tutorial](https://vercel.com/guides/nextjs-prisma-postgres). 

It all makes sense but the only demonstration of the update function is clicking the publish button.

Is there a tutorial that steps through how to edit the title/content - so that when the edit button is clicked, the form loads showing the values originally created, and allows them to be updated? 

I am slow to learn and getting stuck in trying to apply the principles for publish to this interaction.

**Comment by nurul3101** [comment]
Hello @MincePie 👋 

If you are looking for a more complex example, then I would recommend you to have a look at [prisma-examples](https://github.com/prisma/prisma-examples) repository. There are numerous examples of using prisma with Next.js

I would recommend you to have a look at this [example](https://github.com/prisma/prisma-examples/tree/latest/typescript/rest-nextjs-api-routes) which demonstrates all the basic operations (CRUD).

**Reply by MincePie** [reply]
Thank you @nurul3101 -  the example you suggest does not demonstrate an update form. I asked the question because I cna't find an example that does.

If you know of an example that does demonstrate the edit feature, I'd be very grateful for the reference.

**Reply by nurul3101** [reply]
If you are looking for an example which updates a form then you might want to have a look at this [blog](https://daily.dev/blog/nextjs-with-prisma).
It has a section for updating [here](https://daily.dev/blog/nextjs-with-prisma#editfood).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
